ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/composables/PaywallIcon.kt

```diff
@@ -0,0 +1,169 @@
+package com.revenuecat.purchases.ui.revenuecatui.composables
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.aspectRatio
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.lazy.grid.GridCells
+import androidx.compose.foundation.lazy.grid.LazyVerticalGrid
+import androidx.compose.material.Icon
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.revenuecat.purchases.ui.revenuecatui.R
+import kotlin.random.Random
+
+@Composable
+internal fun PaywallIcon(
+    icon: PaywallIconName,
+    modifier: Modifier = Modifier,
+    tintColor: Color,
+) {
+    Icon(
+        modifier = Modifier
+            .aspectRatio(1.0f)
+            .fillMaxSize()
+            .then(modifier),
```

Comment:
So this would apply the modifiers in the `modifier` parameter AFTER the aspectRation/fillMaxSize modifiers. That might be ok depending on the usage. Most times, when there is a `modifier` parameter, I've seen it used instead of the `Modifier` name, so this would look like:
```
        modifier = modifier
            .aspectRatio(1.0f)
            .fillMaxSize(),
```
But again, this might be correct if we need to apply the modifiers in the parameters last.